tests/Unit/CssInlinerTest.php

```diff
@@ -445,12 +445,31 @@ public function wbrTagDataProvider()
      *
      * @dataProvider wbrTagDataProvider
      */
-    public function inlineCssByDefaultRemovesWbrTag($html)
+    public function inlineCssByDefaultNotRemovesWbrTag($html)
```

Comment:
I propose we drop the negation to reduce the cognitive load when reading the code: NotRemoves -> Keeps

Reply:
… or "ByDefaultKeeps"

Reply:
Done.